test/test_poolmanager.py

```diff
@@ -13,30 +14,22 @@
 )
 from urllib3.util import retry, timeout
 
-if sys.version_info >= (2, 7):
-    import unittest
-else:
-    import unittest2 as unittest
-
 
-class TestPoolManager(unittest.TestCase):
+class TestPoolManager(object):
     def test_same_url(self):
         # Convince ourselves that normally we don't get the same object
         conn1 = connection_from_url('http://localhost:8081/foo')
         conn2 = connection_from_url('http://localhost:8081/bar')
-        self.addCleanup(conn1.close)
-        self.addCleanup(conn2.close)
```

Comment:
Yep. I ran the test before and after – no warnings either time about unclosed sockets. Additionally, I don’t see any new warnings appearing in CI.